StoreSpidLogs/index.ts

```diff
@@ -1,11 +1,23 @@
 import { Context } from "@azure/functions";
+import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
 import { AzureContextTransport } from "io-functions-commons/dist/src/utils/logging";
 import * as t from "io-ts";
 import { UTCISODateFromString } from "italia-ts-commons/lib/dates";
 import { readableReport } from "italia-ts-commons/lib/reporters";
 import { IPString, PatternString } from "italia-ts-commons/lib/strings";
+import * as NodeRSA from "node-rsa";
 import * as winston from "winston";
 
+const rsaPublicKey = getRequiredStringEnv("RSA_PUBLIC_KEY");
+
+/**
+ * Encrypt a given string with RSA Public key
+ * @param payload: a json payload to encrypt with public key
+ */
+const encryptWithRsaPublicKey = (payload: string, publicKey: string) => {

```

Comment:
just type this inline instead of creating a new method. avoid creating methods that call `new` internally, they're hard to test. another option would be to pass the result of `new NodeRSA` to the method but that would equal to alias `key.encrypt`.

Reply:
Ok, do i have to wrap it into a tryCatch?